.github/workflows/issue-template-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Check Issue Template
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  check_issue:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: pull in script
+        uses: actions/checkout@v4
+        with:
+          repository: felix920506/jellychan
```

Comment:
Should be upstreamed and given a proper name (something like jellyfin-bot-triage?)

Reply:
In a different repo under the Jellyfin organization?

Reply:
this has been sorted